src/drill/drill.js

```diff
@@ -0,0 +1,17 @@
+const React = require('react');
+const omit = require('lodash.omit');
+const { shallow } = require('enzyme');
+
+const shallowRenderProp = (wrapper, expander) =>
+  typeof expander === 'string'
+    ? wrapper.prop(expander)()
+    : expander(wrapper.props());
+
+module.exports = function renderProp(expander) {
```

Comment:
Shouldn't this be `drill`?

Reply:
Yep, this is a copy&paste leftover. Nice catch!